jicoco/src/main/java/org/jitsi/rest/Health.java

```diff
@@ -31,20 +31,19 @@
 @Path("/about/health")
 public class Health
 {
-    @Inject
-    protected HealthCheckServiceSupplier healthCheckServiceSupplier;
+    @NotNull
+    private final HealthCheckService healthCheckService;
+
+    public Health(@NotNull HealthCheckService healthCheckService)
+    {
+        this.healthCheckService = healthCheckService;
+    }
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     public Response getHealth()
     {
-        HealthCheckService healthCheckService = healthCheckServiceSupplier.get();
-        if (healthCheckService == null)
-        {
-            throw new NotFoundException();
-        }
-
-        Exception status = healthCheckServiceSupplier.get().getResult();
+        Exception status = healthCheckService.getResult();
```

Comment:
This makes the health check service mandatory and I _think_ I remember seeing some people in the community who had disabled it, I could be misremembering though.  It does seem reasonable to want to be able to pass null here, I think?

Reply:
The bridge always passes a non-null value, and in jicofo I have this:
```
if (healthChecker != null)
{
    register(new org.jitsi.rest.Health(healthChecker));
}        
```

If we don't register it, it will return 404 as before